client/header.js

```diff
@@ -368,13 +368,16 @@ Template.header_breadcrumb_puzzle.helpers({
 });
 
 Template.header_breadcrumbs.onCreated(function () {
-  this.autorun(() =>
-    Meteor.call("getRinghuntersFolder", function (error, f) {
-      if (error == null) {
-        Session.set("RINGHUNTERS_FOLDER", f || undefined);
-      }
-    })
-  );
+  this.autorun(async function () {
+    if (!Meteor.userId()) {
+      return;
+    }
+    if (Meteor.callInFlight()) {
+      return;
+    }
+    await Meteor.serializeCall("getRinghuntersFolder");
```

Comment:
Need to assign this to f, although given the test failures I should just make it a subscription.